### PR TITLE
Class4 Homework + Bonusssss

### PR DESCRIPTION
**Pull request by sneakymister**

### CODE REVIEW

File: judith_shimer/class3/teddit_hash.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Let's make an array of hashes to represent news stories in Teddit.
+story1 = {
+	"headline" => "Obama Invites Pizza Rat to White House",
+	"category" => "Politics",
+	"upvotes" => 90
+}
+
+story2 = {
+	"headline" => "Everything Is Terrible",
```
Comment (trivett) [inline judith_shimer/class3/teddit_hash.rb]: lol

File: judith_shimer/class3/teddit_hash.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	"category" => "Environment",
+	"upvotes" => 1000
+}
+
+story3 = {
+	"headline" => "God Is Dead",
```
Comment (trivett) [inline judith_shimer/class3/teddit_hash.rb]: lol #2

File: judith_shimer/class4/max_reverse_engineer.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+nums = [376,406,2985704,58765,2938570,58765,580675,76879,3363]
+puts "The array: #{nums}"
+#the easy thing to do would be to print nums.max, but read on!!!
+
+def my_max(array)
+	array.sort.last
```
Comment (trivett) [inline judith_shimer/class4/max_reverse_engineer.rb]: :+1:

File: judith_shimer/class4/tunr.rb [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+    puts "Continue? (y/n)"
+    play = gets.chomp.upcase
+  end
+end
+
+#calls the method, makes the fun happen
+lets_go
+
+__END__
```
Comment (trivett) [inline judith_shimer/class4/tunr.rb]: aha, cool.

**Comment by trivett** [comment]
Well done @sneakymister. Any pressing questions on the first two weeks of material?

**Comment by sneakymister** [comment]
Nope, all cool so far!

On Sun, Nov 15, 2015 at 10:49 AM, Vincent <notifications@github.com> wrote:

> Well done @sneakymister <https://github.com/sneakymister>. Any pressing
> questions on the first two weeks of material?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/trivett/BEWD-NYC/pull/30#issuecomment-156821879>.
>



-- 
The Sneaky Mister
(Judith)

**Pull request merged**